packages/editor/src/components/editor-canvas/style.scss

```diff
@@ -1,5 +1,6 @@
 .editor-canvas__iframe {
 	&.has-history {
 		padding: $grid-unit-60 $grid-unit-60 0;
+		background-color: $gray-900;
```

Comment:
This color already exists for `,edit-post-visual-editor`:

https://github.com/WordPress/gutenberg/blob/81852a55e700afdaaca7657a2382dd38c0e52b5a/packages/edit-post/src/components/visual-editor/style.scss#L14-L14

Could we somehow leverage that?

Or switch off the `#fff` background color for `iframe[name=editor-canvas]` using the `has-history` class. Here: https://github.com/WordPress/gutenberg/blob/1b2dac78b60c4bab6d1dcb771498d99fbea2ff54/packages/block-editor/src/components/block-canvas/style.scss

That way we don't need this new CSS file.

Reply:
Here's this PR using a theme (empty theme) that doesn't specify a background color:

https://github.com/WordPress/gutenberg/assets/6458278/50772e8a-6cce-4954-90af-9f984f419d5b

Maybe not as important as the issue raised in https://github.com/WordPress/gutenberg/pull/57330 so long as the pattern is visible?

Reply:
Thanks @ramonjd, how does [this look ](https://github.com/WordPress/gutenberg/pull/57631/commits/cc615a8d643ef81819b44f6b9d7a55a9bb114b9b)as an alternative?

Reply:
I think it achieves what we want. I can't see any regressions from https://github.com/WordPress/gutenberg/pull/57330 👍🏻